javaslang/src/test/java/javaslang/collection/LinkedHashMapTest.java

```diff
@@ -69,8 +69,14 @@ protected String className() {
 
     @Test
     public void shouldKeepOrder() {
-        CharSeq actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(CharSeq.empty(), (s, t) -> s.append(t._2));
-        assertThat(actual).isEqualTo(CharSeq.of("abc"));
+        List<Character> actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(List.empty(), (s, t) -> s.append(t._2));
+        Assertions.assertThat(actual).isEqualTo(List.of('a', 'b', 'c'));
```

Comment:
Would it be simpler to have String instead of List?

```java
String actual = LinkedHashMap....mkString();
Assertions.assertThat(actual).isEqualTo("abc");
```